colcon_ros_cargo/package_identification/ament_cargo.py

```diff
@@ -22,17 +23,25 @@ def __init__(self):  # noqa: D107
             '^1.0')
 
     def identify(self, metadata):  # noqa: D102
-        if metadata.type is not None and metadata.type != 'ament_cargo':
-            return
+        from catkin_pkg.package import parse_package
```

Comment:
Can we just keep this with the rest of the imports? We don't really do this "import in smallest scope" throughout the rest of the codebase

Reply:
9439dd1188725caf6fb65a29a04eb4c5aa4e2551